source/jormungandr/jormungandr/planner.py

```diff
@@ -62,15 +64,17 @@ def __init__(self, instance):
     def journeys(self, origins, destinations, datetime, clockwise, journey_parameters, bike_in_pt):
         req = request_pb2.Request()
         req.requested_api = type_pb2.pt_planner
-        for stop_point_id, access_duration in origins.items():
-            location = req.journeys.origin.add()
-            location.place = stop_point_id
-            location.access_duration = access_duration
-
-        for stop_point_id, access_duration in destinations.items():
-            location = req.journeys.destination.add()
-            location.place = stop_point_id
-            location.access_duration = access_duration
+        if origins:
```

Comment:
Because of https://github.com/AurelienLP/navitia/pull/1/files#diff-a71956bda5e0a27a0e996b55cf33ddeaR152
But there is two problems :
1) I should not send None but []
2) I should send either origin fallback or destination fallback

So I shouldn't test here